src/opcodes.rs

```diff
@@ -553,7 +554,57 @@ lazy_static! {
             .with_rmw()
             .with_imm8()
             .into_table(&mut ops);
-        
+        // Begin cmp opcodes
+        //0x38 cmp r/m8, r8
+        define_opcode(0x38).calls(sub_8bit).with_gas(Low)
```

Comment:
I assume a copy paste error here, where `sub` is used instead of `cmp` for the logic